### ISSUE TITLE
Documentation build failing with Sphinx 1.8.0

### ISSUE THREAD

**Issue by fentonscode** [alan-turing-institute/mogp_emulator#93]
**Description**

When attempting to render the documentation to HTML an exception is raised from Sphinx

**To reproduce**

Run the command `make html` from within the docs directory

Console output is as follows:
<pre>
Running Sphinx v1.8.0
making output directory...
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 205 source files that are out of date
updating environment: 205 added, 0 changed, 0 removed
reading sources... [  0%] DimensionReduction
Exception occurred:
  File "\python37\lib\site-packages\sphinx\util\__init__.py", line 318, in get_module_source
    eggpath, _ = re.split('(?<=\\.egg)/', filename)
ValueError: not enough values to unpack (expected 2, got 1)
The full traceback has been saved in C:\Users\cf502\AppData\Local\Temp\sphinx-err-qjolfbpc.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
make: *** [html] Error 2
</pre>

**Environment:**
 - OS: Windows 10 x64
 - Python: 3.7.4
 - Version:  0.2.0 Commit ID 58f6c2eb70d56efa432abd5af163264beed5cc56

**Comment by edaub** [comment]
Thanks for noting this, there are a few documentation build problems that I am currently working on (I have not had much time to dedicate to this project over the past few months). I haven't tested on Windows before, but will see if I can reproduce with the package versions you have on my own.

If you need to consult the documentation for this particular version, you can find it [here](https://mogp-emulator.readthedocs.io/en/v0.2.0/).

**Comment by fentonscode** [comment]
Thanks Eric, I will take a look there. I can submit fixes (if time allows) from my end if that would be helpful?

**Comment by edaub** [comment]
Yes, if you do find the problem and want to submit a fix that would be great. I have been meaning to add contributing guidelines to the repo but have not gotten around to doing so, so please let me know if you have any questions or if there is something I can help with. Thanks!
